gemfiles/rails5.2.gemfile

```diff
@@ -1,3 +1,4 @@
 eval_gemfile 'common.rb'
 
-gem 'activerecord', '~> 5.2.0.beta2'
+gem 'activerecord', '5.2.0.beta2'
```

Comment:
@pschambacher Would you try making a PR using 5.2.0.rc2? I think I saw failing tests…

Reply:
I tried. They completely refactored the migration mechanism in rc1 and haven't had the time to sit and find what's up exactly.

Reply:
This commit sounds like a nice change though: https://github.com/rails/rails/commit/a2827ec9811b5012e8e366011fd44c8eb53fc714

Found by `git bisect` as the commit that breaks the build :-)

Reply:
Yeah probably, just wished it happened before beta2/rc1 😄